components/Modules/modules-content.tsx

```
@@ -84,6 +84,36 @@ export default function ModulesPageContent({
+  const filterResources = useCallback(
+    (resources: Resource[], query: string) => {
+      if (!query) {
+        setFilteredResources(resources);
+      } else {
+        const filtered = resources.filter(
+          (resource) =>
+            resource.title.toLowerCase().includes(query.toLowerCase()) ||
+            (resource.description &&
+              resource.description
+                .toLowerCase()
+                .includes(query.toLowerCase())) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)